sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/pythonLogicalOperators.scala

```diff
@@ -171,6 +186,18 @@ case class ArrowEvalPython(
     copy(child = newChild)
 }
 
+/**
+ * A logical plan that evaluates a [[PythonUDTF]].
+ */
+case class BatchEvalPythonUDTF(
```

Comment:
Can we use Arrow? We plan to migrate to Arrow by default. It's not worthwhile refactoring the batch eval code path ..

Reply:
Yes batch eval is the baseline impl and we should enable arrow by default for UDTFs. I am going to work on it next given we are supporting complex types for arrow based UDFs.

> It's not worthwhile refactoring the batch eval code path 

@HyukjinKwon that's a good point. Should I consider duplicating the code instead of refactoring it?

Reply:
Actually, my point was to don't have batch version but Arrow version alone (since that's going to be the default soon). But okay since the work is already done here.